### ISSUE TITLE
validate public key on payments

### ISSUE THREAD

**Issue by yegor256** [zerocracy/zold#68]
We don't do that now

**Comment by yegor256** [comment]
@rultor release, tag is `0.9.5`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@6a151509-d0d4-40fa-a569-53d8cdf9ca67, thread: pool-246-thread-1
com.zerocracy.pm.Claims[113] java.lang.IllegalStateException: Duplicate claims are not allowed in [C91QJT4CF](https://www.0crat.com/p/C91QJT4CF), can't add this XML to 2 existing ones (Close job;{job=gh:zerocracy/zold#68, reason=GitHub issue was closed},Close job;{job=gh:zerocracy/zold#68, reason=GitHub issue was closed}):
<?xml version="1.0" encoding="UTF-8"?>
<claim id="1471515554">
   <created>2018-05-27T15:09:39.171Z</created>
   <type>Close job</type>
   <token>github;zerocracy/zold;68</token>
   <author>yegor256</author>
   <params>
      <param name="job">gh:zerocracy/zold#68</param>
      <param name="reason">GitHub issue was closed</param>
   </params>
   <until>2018-05-27T15:24:39.172Z</until>
</claim>

```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm): Issue: zerocracy/zold#68, Action: `closed`
